packages/channels/botpress-channel-web/src/views/web/convo/index.jsx

```diff
@@ -127,32 +130,32 @@ export default class Convo extends React.Component {
   renderWelcomeMessage() {
     const message = this.props.config.welcomeMsgText || DEFAULT_WELCOME_MESSAGE
 
-    return <div className={style.paragraph}>{message}</div>
+    return <div className={`${style.paragraph} botpress__paragraph`}>{message}</div>
```

Comment:
Please remember that _unlike_ CSS Modules that are already _scoped_ to the specific file these hook classes are _global_ so let's be more specific. Like `botpress__convo-paragraph`, and similarly for header, line (what is line?), title and others.